File: devops-scm-api/src/main/java/com/tencent/devops/scm/api/pojo/Milestone.java
package com.tencent.devops.scm.api.pojo;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Milestone {
    private Integer id;
    private String title;
    private String state;
    private Integer iid;
    private Date dueDate;
    private String description;
}

File: devops-scm-api/src/main/kotlin/com/tencent/devops/scm/api/pojo/Milestone.kt
package com.tencent.devops.scm.api.pojo

import java.time.LocalDateTime

data class Milestone(
    var id: Int,
    var title: String,
    var state: String? = null,
    var iid: Int? = null,
    var dueDate: LocalDateTime? = null,
    var description: String? = null
)
